HLTrigger/Egamma/src/HLTElectronEtFilter.cc

```diff
@@ -26,17 +26,15 @@
 // constructors and destructor
 //
 HLTElectronEtFilter::HLTElectronEtFilter(const edm::ParameterSet& iConfig) : HLTFilter(iConfig) {
-  candTag_ = iConfig.getParameter< edm::InputTag > ("candTag");
-  EtEB_ = iConfig.getParameter<double> ("EtCutEB");
-  EtEE_ = iConfig.getParameter<double> ("EtCutEE");
+  candTag_   = iConfig.getParameter< edm::InputTag > ("candTag");
+  EtEB_      = iConfig.getParameter<double> ("EtCutEB");
+  EtEE_      = iConfig.getParameter<double> ("EtCutEE");
 
   ncandcut_  = iConfig.getParameter<int> ("ncandcut");
-  doIsolated_ = iConfig.getParameter<bool> ("doIsolated");
 
-  L1IsoCollTag_= iConfig.getParameter< edm::InputTag > ("L1IsoCand");
-  L1NonIsoCollTag_= iConfig.getParameter< edm::InputTag > ("L1NonIsoCand");
+  l1EGTag_   = iConfig.getParameter< edm::InputTag > ("l1EGCand");
```

Comment:
This new python config parameter ```l1EGCand``` is missing in ```fillDescriptions()```
Since this filter is currently not used, a fix can come later in separate PRs for 80X and 81X.